### PR TITLE
fix(UI) : UI try to get resources even if /monitoring/resources is not allowed

### PR DESCRIPTION
**Pull request by Exodiaaaa**
## Description

Fix UI so he dont get resources even if /monitoring/resources is not allowed

**Fixes** # (MON-21880)

## Type of change

- [X] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [X] 23.10.x (master)

<h2> How this pull request can be tested ? </h2>

- Create a user who has access to Centreon UI but without acl (no admin)

- Open Browser console (F12) and switch to network tab

- Login to Centreon with the acl user

- Go to resources status page

result :
UI fetches resources API only if page is allowed

#### Community contributors & Centreon team

- [X] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [X] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/www/front_src/src/Resources/Listing/useLoadResources/index.ts [hunk 1]
```diff
@@ -277,6 +277,10 @@ const useLoadResources = (): LoadResources => {
   };
 
   useEffect(() => {
+    const user = useAtomValue(userAtom);
+    if(!user.is_admin){
+      return ;
+    }
```
Comment (Thebarda) [inline centreon/www/front_src/src/Resources/Listing/useLoadResources/index.ts:283]: So if you are not an admin, you cannot load resources status, right?

Comment (Exodiaaaa) [reply centreon/www/front_src/src/Resources/Listing/useLoadResources/index.ts]: yes that's it

Comment (Med1233) [reply centreon/www/front_src/src/Resources/Listing/useLoadResources/index.ts]: @Exodiaaaa  Consider using navigationAtom to verify the existence of a page with label : 'Resources Status'. you can use some functions from https://ramdajs.com/docs/ to achieve this

Comment (Thebarda) [reply centreon/www/front_src/src/Resources/Listing/useLoadResources/index.ts]: I think you would consider manage this in the react router

**Review by Thebarda — CHANGES_REQUESTED** [review]

**Review by Thebarda — APPROVED** [review]

**Review by Med1233 — APPROVED** [review]

**Pull request merged**